docs/advanced-concepts/route-narrowing.md

```diff
@@ -50,7 +50,7 @@ You can also use the `isRoute` type guard. You could write the same logic as abo
 
 ```ts
 if(router.route, 'user', { exact: true }) {
-  router.route.key // "user"
+  router.route.name // "user"
```

Comment:
I like that there's one less concept ❤️ route.name reads so nicely

Reply:
agreed 👍